ragstack-e2e-tests/pyproject.llamaindex.toml

```diff
@@ -24,7 +24,7 @@ nemoguardrails = "^0.8.0"
 beautifulsoup4 = "^4"
 
 llama-index = { git = "https://github.com/run-llama/llama_index.git", branch = "main" }
-llama-index-readers-llama-parse = { git = "https://github.com/run-llama/llama_index.git", branch = "main", subdirectory = "llama-index-integrations/readers/llama-index-readers-llama-parse" }
+llama-index-readers-llama-parse = { git = "https://github.com/nicoloboschi/llama_index.git", branch = "llama-parse", subdirectory = "llama-index-integrations/readers/llama-index-readers-llama-parse" }
```

Comment:
See https://github.com/run-llama/llama_index/pull/12232

Reply:
ah, I did the same PR then https://github.com/run-llama/llama_index/pull/12236